File: csharp/test/ExistsTest.cs
// BluefinTecsMerchantPortal SDK exists test.

using Xunit;

using BluefinTecsMerchantPortalSdk;

namespace BluefinTecsMerchantPortalSdk.Test;

public class ExistsTest
{
    [Fact]
    public void TestMode()
    {
        var testsdk = BluefinTecsMerchantPortalSDK.TestSDK(null, null);
        Assert.NotNull(testsdk);
    }
}

File: rb/test/exists_test.rb
# BluefinTecsMerchantPortal SDK exists test

require "minitest/autorun"
require_relative "../BluefinTecsMerchantPortal_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = BluefinTecsMerchantPortalSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end
